StripeCore/StripeCore/Source/Attestation/StripeAttest.swift

```diff
@@ -117,33 +119,42 @@ import UIKit
         case successfullyAttested = "STPAttestKeySuccessfullyAttested"
     }
 
-    private static var keyID: String? {
+    private var keyID: String? {
```

Comment:
keyID is used way too many times in this file, can we rename this to something like `storedKeyID` ?